### PR TITLE
[Needs re-review] Add permissions to release milestone step plugin

### PR DESCRIPTION
**Pull request by svanoort**
Review by @jglick or @amuniz as current committers

### CODE REVIEW

**Comment by amuniz** [comment]
👍

**Comment by amuniz** [comment]
The minimum needed is me then. I can cut the release if needed.

File: permissions/plugin-pipeline-milestone-step.yml [hunk 1]
```diff
@@ -4,4 +4,9 @@ paths:
 - "org/jenkins-ci/plugins/pipeline-milestone-step"
 developers:
 - "amuniz"
+- "svanoort"
+- "abayer"
 - "jglick"
+- "rsandell"
+- "stephenc"
```
Comment (batmat) [inline permissions/plugin-pipeline-milestone-step.yml]: :-1: 

Beware, it's the Jenkins LDAP/JIRA usernames here, not GitHub.

Use https://issues.jenkins-ci.org/secure/ViewProfile.jspa?name=stephenc URL to check.

@stephenc is https://issues.jenkins-ci.org/secure/ViewProfile.jspa?name=stephenconnolly

File: permissions/plugin-pipeline-milestone-step.yml [hunk 2]
```diff
@@ -4,4 +4,9 @@ paths:
...
+- "oleg-nenashev"
```
Comment (daniel-beck) [inline permissions/plugin-pipeline-milestone-step.yml]: And Oleg is `oleg_nenashev`.

Sooooo many existing YAML files to copy user names from…

File: permissions/plugin-pipeline-milestone-step.yml (hunk 1 above)
Comment (svanoort) [inline permissions/plugin-pipeline-milestone-step.yml]: @batmat  Oopsy, that's a gotcha!  Lesson learned about trying to do more than the minimum, especially when trying to ship fixes on a timeline.

File: permissions/plugin-pipeline-milestone-step.yml (hunk 2 above)
Comment (svanoort) [inline permissions/plugin-pipeline-milestone-step.yml]: @daniel-beck Addressed.

**Pull request merged**